src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -404,16 +400,14 @@ protected function runMigration($migration, $method)
     /**
      * Pretend to run the migrations.
      *
-     * @param  object  $migration
-     * @param  string  $method
+     * @param  string  $migrationName
+     * @param  string[]  $queries
      * @return void
      */
-    protected function pretendToRun($migration, $method)
+    protected function pretendToRun($migrationName, $queries): void
```

Comment:
Please remove all return types. Those are breaking changes to a method signature.

Reply:
Also, the behavior of this method changed. You can't change the parameter types or what this method does. Please revert all changes to existing methods so they behave like before.

Reply:
This method's behavior was restored.